### ISSUE TITLE
Windows 11: NVDA does not redirect focus to the current element in the system tray when pressing escape.

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#14857]
### Steps to reproduce:
1. Open NVDA
2. Navigate to the system tray
3. press enter on "Show hidden symbols" I am not sure how it is called in english since my Windows is in German
4. focus any of the icons, press context menu or shift+f10 and then escape to exit the context menu

### Actual behavior:
For the icons that are not hidden like clock, batery status etc. everything is working well and NVDA redirects the focus to the current button on the system tray after closing its context menu
However, for the elements that have been expanded and were hidden before, this ndoes not work. the focus is redirected to some cryptical window class names and then NVDA is silent when pressing arrow keys or tab key.
In my case:
* Context menu on NVDA and then escape redirects me to a Malwarebytes Icon which is not reachable
* Context menu on Powertoys button and then escape redirects me to a window "PToy Tray Icon Window" which is not reachable
* Context menu on NVIDIA Geforce GTX 3070 TI Laptop GPU redirects me directly to a window "Optimus Icon Window" which is not reachable. From here I even cannot press escape to land again on the system tray.
* Context menu on NVIDIA settings button redirects me to a window "Notification icon window", from here I cannot even press escape to land again on the system tray.
* Context menu on Bluetooth devices button and then escape redirects me to a Window Class "Bluetooth notification aria icon window class"
It impacts almost every icon on the system tray.
In case of NVIDIA GPU icons, I think there is even a menu appearing when pressing shift+f10 on the icon, but NVDA is not redirected to that menu.

### Expected behavior:
NVDA should land on the button of the icon on the system tray after closing its context menu.
For NVIDIA GPU icons, NVDA should be redirected to the menu which is appearing on the screen when pressing shift+f10 or right click on the icon in the system tray.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: alpha-28126,03c31e5b (2023.2.0.28126)

#### Windows version:
Windows 11 22 H2 update

#### Name and version of other software in use when reproducing the issue:
n/a

#### Does the issue still occur after restarting your computer?
yes

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
I can test only fin Windows 11, but if some one can provide me a full build of Windows 10, i can also include that in my reporting to Microsoft.
It seems it occurs with Jaws and Narator as well.

**Comment by Adriani90** [comment]
Reported to microsoft accessibility desk,
ticket number is 1052626026.

**Comment by alexstine** [comment]
Any update? I could reproduce this today on my new Windows 11 work laptop. :(

**Comment by seanbudd** [comment]
@alexstine - you'll need to contact microsoft to get an update

**Comment by alexstine** [comment]
@seanbudd Thanks, thought I'd ask here first before my request falls into a deep dark hole that is the Microsoft Disability Answer Desk. I'll write up a fresh report tomorrow.

**Comment by Adriani90** [comment]
Note this occurs only on certain icons on the system tray, e.g. focusing the NVDA icon with the keyboard and pressing right klick on it followed by escape sends the focus to the task bar instead of the NVDA icon.

Doing the same on e.g. powertoys sends the focus to an invalid control or something, at least it is not interactive.

I guess the focus redirection needs to be specified by the vendors of the affected applications, but it might be also solved globally in Windows.

cc: @msftedad do you have any insights to which team we should report this? I have tried several times via the Microsoft accessibility desk but I was not successful. Could you report this issue to the coresponding team?

**Comment by msftedad** [comment]
@Adriani90 Issue is shared with Windows team and we will keep you updated once we have more information to share.

**Comment by msftedad** [comment]
Hi @Adriani90, Issue shared with dependent team, for reference, team have created a ticket "[Accessibility 229717](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2FCSS-IRT%2FCSS%2520IRT%2F_workitems%2Fedit%2F229717&data=05%7C02%7Cv-ymounika%40microsoft.com%7Cb3a472e983b842d0a5d708dd0ac672dd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638678569637207428%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=HjzE0yUsntqfl%2BNkNOHvUAUtVfQh82eLV6Jk5O04byA%3D&reserved=0)" tracking purposes, you won’t have access to this link, but this can be used as a reference number for this request. Thanks
